samples/SpectatorView.Example.Unity/README.md

```diff
@@ -1,3 +1,70 @@
+
+> **ARKit Note:** If you wish to run the experience on an iOS device with ARKit, download the [ARKit repository](https://bitbucket.org/Unity-Technologies/unity-arkit-plugin/downloads/), unzip it, and copy the contents of the `Unity-Technologies-unity-arkit-plugin-94e47eae5954/Assets` folder to `/samples/Build2019Demo.Unity/Assets/`.
+
+### Instructions
+
+These instructions assume that a HoloLens device will be used to host the shared experience and that an Android phone will be used to spectate the shared experience. For iOS, replace references to Android with iOS.
+
+#### Prepare your local codebase
+
+1. Run `tools/Scripts/SetupRepository.bat` (`tools/Scripts/SetupRepository.sh` on Mac) to setup your local version of the code base. This script will obtain the external dependencies required for building this project.
+2. If you are using a HoloLens 1 device, build the `SpectatorView.OpenCV.dll` plugin and include it in the SpectatorView.Unity codebase. Instructions for building and adding `SpectatorView.OpenCV.dll` to the Spectator View codebase can be found [here](../../src/SpectatorView.Native/README.md).
+
+#### Prepare your devices
+
+1. Connect your devices to the same WiFi network.
+2. Obtain and write down your HoloLens's IP Address (This can be found in the settings menu).
+
+#### Configure your project
+
+1. Open the `SpectatorView.Example.Unity` project in Unity.
+2. If building for HoloLens 2, ensure that `QRCODESTRACKER_BINARY_AVAILABLE` has been defined as a Scripting Define Symbols in your WSA Unity Player Settings. If you have made no changes to the SpectatorView.Examples.Unity project definition, this should already be defined (Player settings can be found under `Build Settings -> Player Settings`).
+3. If building for HoloLens 1, ensure that `SpectatorView.OpenCV.dll` exists in the project's Assets folder. It should have been placed under Assets\MixedReality-SpectatorView\SpectatorView.Native\Plugins\WSA\x86 when going through the compilation instructions [here](../../src/SpectatorView.Native/README.md).
+
+#### Build & Deploy
+
+##### HoloLens
+
+1. Select `HoloLens` as your target platform using Spectator View's `PlatformSwitcher` (This can be found in the Unity inspector for the `SpectatorView` prefab GameObject).
+2. Build UWP Player containing `MixedReality-SpectatorView/SpectatorView/Scenes/SpectatorView.HoloLens` scene, and deploy this application to the HoloLens device.
+3. Launch `SpectatorView.Example.Unity` on the HoloLens and wait for the experience to start.
+
+![Marker](./../../doc/images/HoloLensSpectatorViewExampleBuildSettings.png)
+
+##### Android
+
+1. Select `Android` as your target platform using Spectator View's `PlatformSwitcher` (This can be found in the Unity inspector for the `SpectatorView` prefab GameObject).
+2. Include and declare `MixedReality-SpectatorView/SpectatorView/Scenes/SpectatorView.Android.unity` as Scene 0 in your Unity Build Settings.
```

Comment:
For my own reference in the future does the word 'declare' mean "make sure it's checked" (and is this language commonly known for all folks doing Unity stuff?)

Reply:
yeah, let me change this to check.

---
In reply to: [352946072](https://github.com/microsoft/MixedReality-SpectatorView/pull/290#discussion_r352946072) [](ancestors = 352946072)